### PR TITLE
[config] adopters. address issue #571

### PR DESCRIPTION
**Pull request by pabloopez**
Signed-off-by: pablopez <pablo.lopezzaldivar@sysdig.com>

**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

No area, this does not affect configuration. But please let's double check that there are no breaking changes. After this changes I was able to deploy locally with no errors the whole webpage with `hugo server`.

**What this PR does / why we need it**:

This is adding a little bit of order to the `config.toml` file. Adopters and training contributors are stored in a separate `params.toml` file now.

**Which issue(s) this PR fixes**:

#571 

Fixes #

**Special notes for your reviewer**:

This was discussed with @leogr.

### CODE REVIEW

File: config/_default/config.toml [hunk 1]
```diff
@@ -389,165 +389,7 @@ url = "https://falco.org"
 
 # Training resources provided by the community
 # If you are adding a new resource, remember to include an image in static/img/training/
```
Comment (leogr) [inline config/_default/config.toml:391]: This comment does not make sense anymore here.

**Review by leogr — COMMENTED** [review]
I'd also update the "Add new resources" link from :point_down: 
https://github.com/falcosecurity/falco-website/blob/d28fb098b0c87bf3202f7a3fd139262afe6751fa/layouts/partials/training.html#L42

to 

```
https://github.com/falcosecurity/falco-website/edit/master/config/_default/params.toml
```

Otherwise, SGTM!

Thank you :)

**Comment by pabloopez** [comment]
done, thx for the review @leogr !

**Review by leogr — APPROVED** [review]
LGTM, now!

Thank you :pray: 

/approve

**Pull request merged**